cmd/flags.go

```diff
@@ -37,4 +38,10 @@ var (
 			"Example: <protocol>://<ip>:<port>, tcp://127.0.0.1:26657",
 		DefValue: "",
 	}
+	mutualPeersFlag = &flag.Flag{
+		Name: "p2p.mutual",
+		Usage: "Comma-separated multiaddresses of mutual peers to keep an unbreakable connection with. " +
```

Comment:
"unbreakable connection" isn't really that descriptive, does it just mean that the peers re-dial each other if the connection breaks? Maybe mention that.

Reply:
Unbreakable means it is immune to peer scoring slashing and trimming by connection manager, I.e. Prioritized. Sorry for the confusion. 

So your guess is a bit wrong, but still valid as they try to reconnect to each other if having troubles

Reply:
If the reader guesses wrong, that's indicative of a need for more docs! Can you improve the wording here to include what you just wrote @Wondertan?

Reply:
True, adding more info.